apps/framework-cli/src/cli/local_webserver.rs

```diff
@@ -68,18 +71,13 @@ pub struct RouterRequest {
     route_table: &'static RwLock<HashMap<PathBuf, RouteMeta>>,
 }
 
-#[derive(Serialize, Deserialize, Debug, Clone)]
-enum Direction {
-    In,
-    Out,
-}
-
 #[derive(Serialize, Deserialize, Debug, Clone)]
 pub struct FlowMessages {
```

Comment:
you could remove this all together right? it is the same as `StreamingFunctionEvent`